agent/config/runtime_test.go

```diff
@@ -1633,10 +1633,11 @@ func TestLoad_IntegrationWithFlags(t *testing.T) {
 		expectedWarnings: []string{`The 'acl_datacenter' field is deprecated. Use the 'primary_datacenter' field instead.`},
 	})
 	run(t, testCase{
-		desc: "acl_replication_token enables acl replication",
```

Comment:
It might make sense to create a non-deprecated version of this test to verify that setting `acl.tokens.replication` does NOT enable token replication? Mostly to verify that the deprecated field logic doesn't apply to new configs, just old configs.

Reply:
good call, I'll add that